docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex), [Rainer Stropek](https://www.timecockpit.com), and [Daniel Roth](https://github.com/danroth27)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Publishing the app
+
+Blazor apps are published for deployment in Release configuration with the [dotnet publish](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish) command. An IDE may handle executing the `dotnet publish` command automatically using its built-in publishing features, so it might not be necessary to manually execute the command from a command prompt depending on the development tools in use.
+
+```console
+dotnet publish -c Release
+```
+
+`dotnet publish` triggers a [restore](https://docs.microsoft.com/dotnet/core/tools/dotnet-restore) of the project's dependencies and [builds](https://docs.microsoft.com/dotnet/core/tools/dotnet-build) the project before creating the assets for deployment. As part of the build process, unused methods and assemblies are removed to reduce app download size and load times. The deployment is created in the */bin/Release/\<target-framework>/publish* folder.
+
+The assets in the *publish* folder are deployed to the web server. Deployment might be a manual or automated process depending on the development tools in use.
+
+There are two deployment models for Blazor apps:
```

Comment:
Since the standalone section has the detailed discussion on SPA routing and specifying the app base path I think it would make sense to flip the order here and put the standalone section first. That way when these concepts are referred to in the ASP.NET Core section you know what they are.

Reply:
I think we should reorganize ... it breaks the topic a bit to split the standalone content across the hosted Blazor content.

How about going to this outline:

* Publishing
* Rewrite URLs for correct routing
* App base path
* Deployment types
   * Hosted
   * Standalone with deployment strategies